lib/db/filters/CollisionFiltersSql.js

```diff
@@ -50,6 +51,11 @@ function getCollisionFilters(features, collisionQuery) {
       .join(' OR ');
     filters.push(`(${filterDetails})`);
   }
+  if (sources.length > 0) {
+    params.sources = sources;
+    const filterSources = sources.map(({ field }) => `'${field}'`).join(', ');
```

Comment:
I assume there's no stuff inside the value here, like a `'` or `,` that would bung-up the sql query.